src/utils/synchronize/check.ts

```diff
@@ -65,7 +65,7 @@ const getResult = (
   queenError: string,
   pearlMissing: string[] = [],
   queenMissing: string[] = [],
-  pearlSurveyUnits = [],
+  pearlSurveyUnits: string[] = [],
```

Comment:
shouldn't pearlSurveyUnits be of type SurveyUnit[] ?

Reply:
According to the code, this is an array of id (so an array of string)